src/lib.ts

```diff
@@ -688,6 +692,18 @@ export function getTodaysHolidayWanderers(): Monster[] {
     .flat();
 }
 
+/**
+ * Determines & returns whether or not we can safely call visitUrl(), based on whether we're in a fight, multi-fight, choice, etc
```

Comment:
good and useful function. docstring looks good. 

my only thought is: have you tested this? i see no obvious reason why this shouldn't work; to talk through the logic, a true in any of the conditions examined means that the X or Y or Z has spawned a true, which gets converted to false by the not operator. does current round default to 0? i could see it (possibly) defaulting to something weird instead but you probably have tested it.

Reply:
yes! it does

Reply:
And it works even if you win init